_15thnight/queue.py

```diff
@@ -35,10 +34,28 @@ def queue_send_message(email, number, subject, body):
     """
     if number:
         send_sms(to_number=number, body=body)
-    #send_email(email, subject, body)
     message = Message(body=body, subject=subject, recipients=[email])
     mailer.send(message)
 
+
 @celery.task
 def queue_send_email(message):
-    mailer.send(message)
\ No newline at end of file
+    mailer.send(message)
+
+
+@celery.task
+def queue_send_alert_resolution_notice(need):
+    # Check if alert is closed and trigger an alert closed notification
+    if not need.alert.is_open():
+        alert = need.alert
+        body = (
+            "Alert sent at {0} for a {1}yo {2} with {3} needs has been closed."
+            "\nSee: {4}" % (
+                alert.created_at, alert.age, alert.gender, alert.need_count(),
```

Comment:
Gender could be unspecified, in that case we wouldn't insert it because the original alert would just say `Xy/o`